db_diff/tskdbdiff.py

```diff
@@ -1019,8 +1027,20 @@ def normalize_tsk_objects_path(guid_util: TskGuidUtils, objid: int,
                             path_parts[idx + 2].lower().endswith("report.html"):
                         path_parts = ["Reports", "html-report.html"]
                         break
+            for idx in range(0, len(path_parts) - 1):
+                if path_parts[idx].lower() == "reports" and \
+                        path_parts[idx + 1].lower().startswith("autopsytestcase html report"):
+                    path_parts = ["Reports", "AutopsyTestCase HTML Report"]
+                    break
+                if path_parts[idx].lower() == "reports" and \
+                        "html report" in path_parts[idx + 1].lower() and \
+                        len(path_parts) > idx + 2 and \
+                        path_parts[idx + 2].lower().endswith("report.html"):
+                    path_parts = ["Reports", "html-report.html"]
+                    break
```

Comment:
I think you can take this out since it already happens if it isn't a leapp report.